src/components/ReviewComponents/ReviewComponentsForm.tsx

```diff
@@ -23,16 +23,18 @@ export const ReviewComponentsForm: React.FC<ReviewComponentsFormProps> = ({
   return (
     <Form onSubmit={handleSubmit}>
       {!isSample && (
-        <RadioGroupField
-          isInline
-          name="deployMethod"
-          options={[
-            { label: 'Automatically deploy', value: DeployMethod.AutomaticDeploy },
-            { label: 'Manually deploy', value: DeployMethod.ManualDeploy },
-          ]}
-        />
+        <div className="hacDev-page__section">
+          <RadioGroupField
+            isInline
+            name="deployMethod"
+            options={[
+              { label: 'Automatically deploy', value: DeployMethod.AutomaticDeploy },
+              { label: 'Manually deploy', value: DeployMethod.ManualDeploy },
+            ]}
+          />
+        </div>
       )}
-      <PageSection className="hacDev-review-form__components-section">
```

Comment:
This is much better. I think we can also delete the unused class in ReviewComponentsForm.scss ?

Reply:
Done. Removed the unused css file.